blockmanager.go

```diff
@@ -1879,6 +1879,9 @@ func (b *blockManager) fetchFilterFromAllPeers(
 				// and wait for the next response.
 				filterResponses[sp.Addr()] = gcsFilter
 
+				// close(peerQuit) // this peer responded. don't ask again
```

Comment:
This is what I described in the PR description.  I need input because I never hit this `fetchFilterFromAllPeers` function but from what I can tell it is missing this `close(peerQuit)`